app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -501,11 +500,13 @@ class SectionRenderingService(frontendAppConfig: FrontendAppConfig) {
     hmrcTP: HmrcTaxPeriod) = {
 
     val taxPeriodList = obligations match {
-      case RetrievedObligations(listOfObligations) => listOfObligations.toList
+      case RetrievedObligations(listOfObligations) => listOfObligations
       case _                                       => List[TaxPeriodInformation]()
     }
 
+    import cats.syntax.eq._
```

Comment:
This line is not needed, I can compile the code even when I remove it (as expected).